src/github-api/implementation.ts

```diff
@@ -89,5 +98,36 @@ export function buildGitHubApi(token: string): GitHubApi {
         })
       );
     },
+    loadPullRequestStatus(pr) {
+      const query = gql`
+        query {
+          repository(owner: "${pr.repo.owner}", name: "${pr.repo.name}") {
+            pullRequest(number: ${pr.number}) {
+              reviewDecision
```

Comment:
If I understand the PR correctly, this isn't used yet. Is the intention to replace the current approval filtering logic with this?

Reply:
it's a potential usage. for outgoing PRs, this would be the obvious thing to have, and i'd add that in a separate PR. for incoming, there's still more complex logic, so not sure if we want to immediately want to replace yet.